android-components/components/browser/state/src/test/java/mozilla/components/browser/state/selector/SelectorsKtTest.kt

```diff
@@ -286,4 +286,21 @@ class SelectorsKtTest {
             assertNull(state.findNormalOrPrivateTabByUrl("https://www.mozilla.org", true))
         }
     }
+
+    @Test
+    fun `findNormalOrPrivateTabByUrlIgnoringFragment extension function`() {
```

Comment:
Would it make sense to also add tests for `isSameUrlIgnoringFragment`?

Reply:
That's implicitly tested as part of this and covers the code paths, it's well covered imo 🤔  let me know what you think!

Reply:
No strong opinion here (I'm okay either way) so I will defer to your judgement. The tricky part here was that it was implicit and you had to intentionally match up the `private` parameters with the `Tab` when calling `findNormalOrPrivateTabByUrlIgnoringFragment`. However, I do see `isSameUrlIgnoringFragment` is covered here.